pkg/query/columnquery.go

```diff
@@ -317,6 +317,16 @@ func (q *ColumnQueryAPI) Query(ctx context.Context, req *pb.QueryRequest) (*pb.Q
 		}, nil
 	}
 
+	var functionToFilterBy string
+
+	for _, filter := range req.GetFilter() {
+		if stackFilter := filter.GetStackFilter(); stackFilter != nil {
+			if functionNameFilter := stackFilter.GetFunctionNameStackFilter(); functionNameFilter != nil {
+				functionToFilterBy = functionNameFilter.GetFunctionToFilter()
```

Comment:
this deserves a comment I think, since theoretically the API already allows passing multiple stack filters, but for now we just make the last one actually take effect, which we want to change in the future but not yet

Reply:
makes sense, will add the comment.